flags/src/test/java/com/yahoo/vespa/flags/DimensionTest.java

```diff
@@ -0,0 +1,25 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.flags;
+
+/**
+ * @author hakonhall
+ */
+class DimensionTest {
+    /**
+     * A compile-time test:  If this breaks you have most likely added (or removed) a dimension?
+     * If so you need to update the following:</p>
+     *
+     * <ul>
+     *     <li>Dimension validation in SystemFlagsDataArchive</li>
+     *     <li>Flag edit documenting dimension list in EditFlagDataCommandTest</li>
```

Comment:
This was removed in the internal PR

Reply:
right, good catch, I have fixed this in the next PR